WORKSPACE

```diff
@@ -103,6 +103,16 @@ maven_jar(
    artifact = "net.sf.trove4j:trove4j:3.0.1",
 )
 
+maven_jar(
```

Comment:
We have not publish `tipb` to public maven repo. `maven_jar` assume such repo is already published and will download from web.